src/test/java/underdogs/devbie/answer/acceptance/AnswerAcceptanceTest.java

```diff
@@ -0,0 +1,85 @@
+package underdogs.devbie.answer.acceptance;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import underdogs.devbie.acceptance.AcceptanceTest;
+import underdogs.devbie.answer.dto.AnswerCreateRequest;
+import underdogs.devbie.answer.dto.AnswerResponse;
+import underdogs.devbie.answer.dto.AnswerResponses;
+import underdogs.devbie.answer.dto.AnswerUpdateRequest;
+
+public class AnswerAcceptanceTest extends AcceptanceTest {
+
+    /*
+    Feature: 면접 답변 관리
+     Scenario: 면접 답변을 관리한다.
+         When 질문에 해당하는 답변 1개를 추가 요청한다.
+         Then 답변이 업로드 되었다.
+         When 질문에 해당하는 답변 전체를 조회 요청한다.
+         Then 답변 전체가 조회된다.
+         When 답변 내용을 수정한다.
+         Then 답변이 수정 되었다.
+         When 질문에 해당하는 답변 1개를 조회 요청한다.
+         Then 답변 1개가 조회된다.
+         When 답변을 삭제한다.
+         Then 답변이 삭제되었다.
```

Comment:
```suggestion
    /*
    Feature: 면접 답변 관리
    
     Scenario: 면접 답변을 관리한다.
         When 질문에 해당하는 답변 1개를 추가 요청한다.
         Then 답변이 업로드 되었다.
         
         When 질문에 해당하는 답변 전체를 조회 요청한다.
         Then 답변 전체가 조회된다.
         
         When 답변 내용을 수정한다.
         Then 답변이 수정 되었다.
         
         When 질문에 해당하는 답변 1개를 조회 요청한다.
         Then 답변 1개가 조회된다.
         
         When 답변을 삭제한다.
         Then 답변이 삭제되었다.
```
가독성을 위해 줄을 띄워주는건 어떨까요?